eng/common/instructions/azsdk-tools/typespec-to-sdk.instructions.md

```diff
@@ -17,8 +17,28 @@ Pre-requisites:
 # SDK generation steps
 
 ## Step: Generate SDKs
-**Goal**: Generate SDKs
-**Message to user**: "SDK generation will take approximately 15-20 minutes. Currently, SDKs are generated using the Azure DevOps pipeline. SDK generation is supported only from a merged API spec or from an API spec pull request in the https://github.com/Azure/azure-rest-api-specs repository."
+
+As a first step, you must prompt the user to understand the intention of SDK generation. Based on the user input, you can either run SDK generation locally or use SDK generation pipeline.
```

Comment:
Should we add detail prompts to the user to collect the user intent? For example asking questions like "do you want to customize the SDK code?" The reason is that the following instructions include similar guidance for evaluating the user intent.